CONTRIBUTING.md

```diff
@@ -35,25 +27,51 @@ questions: what changed and why. The subject line should feature the what and
 the body of the commit should describe the why.
 
 ```
-scripts: add the test-cluster command
+Add the test-cluster command
 
-this uses tmux to setup a test cluster that you can easily kill and
+This uses tmux to setup a test cluster that you can easily kill and
 start for debugging.
+```
+
+Commits that fix a Bugzilla bug should add the bug number like `Bug 12345: ` to
```

Comment:
Is this data supposed to be in the first line of the commit message or the description text of the opened PR?